src/utils/api/transaction/lsk.js

```diff
@@ -411,3 +424,20 @@ export const broadcast = async ({ transaction, serviceUrl }) => {
 
   return response;
 };
+
+/**
+ * Computes transaction id
+ * @param {object} transaction
+ * @returns {Promise} returns transaction id for a given transaction object
+ */
+export const computeTransactionId = ({ transaction }) => {
```

Comment:
This doesn't see to return a promise, rather an `id`. Please correct the fn signature document. Please describe the parameters too.